sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Helpers/CSharpSymbolUsageCollector.cs

```diff
@@ -347,12 +331,38 @@ private AccessorAccess EvaluatePropertyAccesses(ExpressionSyntax node)
                 : AccessorAccess.Get;
         }
 
-        private bool IsKnownIdentifier(SimpleNameSyntax nameSyntax) =>
-            knownSymbolNames.Contains(nameSyntax.Identifier.ValueText);
+        private bool HasKnownIdentifier(SimpleNameSyntax nameSyntax) =>
```

Comment:
Can you remove this function and embed it's content on call sites? There are just two.

While I was reading the `VisitIdentifierName` the name `HasKnownIdentifier` made an impression that all subnodes are inspected somehow. Even when the function name is good.